build_tools/benchmarks/comparisons/setup_mobile.sh

```diff
@@ -0,0 +1,164 @@
+
+if [[ ${GPU_TYPE} = "mali" ]]
+then
+echo "Compiling ${MODEL_NAME}.vmfb for vulkan mali..."
+${IREE_COMPILE_PATH} --iree-input-type=tosa --iree-mlir-to-vm-bytecode-module \
+  --iree-hal-target-backends=vulkan-spirv \
+  --iree-vulkan-target-triple=valhall-unknown-android11 \
+  --iree-llvm-debug-symbols=false \
+  --iree-vm-bytecode-module-strip-source-map=true \
+  --iree-vm-emit-polyglot-zip=false \
+  ${IREE_MODEL_DIR}/${MODEL_NAME}.mlir \
+  --o ${IREE_MODEL_DIR}/vulkan/${MODEL_NAME}.vmfb
+else
+echo "Compiling ${MODEL_NAME}.vmfb for vulkan adreno..."
+${IREE_COMPILE_PATH} --iree-input-type=tosa --iree-mlir-to-vm-bytecode-module \
+  --iree-hal-target-backends=vulkan-spirv \
+  --iree-vulkan-target-triple=adreno-unknown-android11 \
+  --iree-llvm-debug-symbols=false \
+  --iree-vm-bytecode-module-strip-source-map=true \
+  --iree-vm-emit-polyglot-zip=false \
+  ${IREE_MODEL_DIR}/${MODEL_NAME}.mlir \
+  --o ${IREE_MODEL_DIR}/vulkan/${MODEL_NAME}.vmfb
+fi
+
+# Cross-compile IREE benchmark binary.
+cd ${SOURCE_DIR}/iree
+cmake -GNinja -B ../iree-build/ \
+  -DCMAKE_INSTALL_PREFIX=../iree-build/install \
+  -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+  .
+cmake --build ../iree-build/ --target install
+
+rm -rf ${SOURCE_DIR}/iree-build-android
+
+cmake -GNinja -B ../iree-build-android/ \
+  -DCMAKE_TOOLCHAIN_FILE="${ANDROID_NDK?}/build/cmake/android.toolchain.cmake" \
+  -DIREE_HOST_BINARY_ROOT="$PWD/../iree-build/install" \
+  -DANDROID_ABI="arm64-v8a" \
+  -DANDROID_PLATFORM="android-29" \
+  -DIREE_BUILD_COMPILER=OFF \
+  .
+cmake --build ../iree-build-android/
+cp ${SOURCE_DIR}/iree-build-android/iree/tools/iree-benchmark-module ${ROOT_DIR}/
+
+# Cross-compile TFLite benchmark binary.
+sudo apt-get install libgles2-mesa-dev
+
+export CC=clang
+export CXX=clang++
+
+cd ${SOURCE_DIR}
+git clone https://github.com/tensorflow/tensorflow.git
+cd tensorflow
+# Select defaults. Answer Yes to configuring ./WORKSPACE for Android builds.
+# Use Version 21 for Android NDK, 29 for Android SDK.
+python configure.py
+bazel build -c opt --config=android_arm64 \
+  --copt="-Wno-error=implicit-function-declaration" \
+  tensorflow/lite/tools/benchmark:benchmark_model
+
+cp ${SOURCE_DIR}/tensorflow/bazel-bin/tensorflow/lite/tools/benchmark/benchmark_model ${ROOT_DIR}/
+
+# Push benchmarking artifacts to device.
+adb shell rm -r /data/local/tmp/mobilebert_benchmarks
+adb push ${ROOT_DIR} /data/local/tmp
+
+DEVICE_ROOT_DIR=/data/local/tmp/mobilebert_benchmarks
+adb shell chmod +x ${DEVICE_ROOT_DIR}/benchmark_model
+adb shell chmod +x ${DEVICE_ROOT_DIR}/iree-benchmark-module
+
+# Setup device.
+adb shell "su root sh ${DEVICE_ROOT_DIR}/setup/set_android_scaling_governor.sh performance"
+
+if [[ ${GPU_TYPE} = "mali" ]]
+then
+adb shell "su root sh ${DEVICE_ROOT_DIR}/setup/set_pixel6_gpu_scaling_policy.sh performance"
+else
+adb shell "su root sh ${DEVICE_ROOT_DIR}/setup/set_adreno_gpu_scaling_policy.sh performance"
+fi
```

Comment:
Nit: formatting (https://google.github.io/styleguide/shellguide.html#loops)

```suggestion
if [[ ${GPU_TYPE} = "mali" ]]; then
  adb shell "su root sh ${DEVICE_ROOT_DIR}/setup/set_pixel6_gpu_scaling_policy.sh performance"
else
  adb shell "su root sh ${DEVICE_ROOT_DIR}/setup/set_adreno_gpu_scaling_policy.sh performance"
fi
```

Reply:
Done